FirebaseAuthUI/FUIPrivacyAndTermsOfServiceView.h

```diff
@@ -30,6 +32,23 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)useFooterMessage;
 
+/** @property authUI
+    @brief The @c FUIAuth instance of the application.
+ */
+@property(nonatomic, strong) FUIAuth *authUI;
```

Comment:
This should be marked as `nullable`, since it may not be set before it's accessed. The comment should describe how it's used:

```objective-c
/** @property authUI
    @brief the @c FUIAuth instance whose bundle will be used to populate the view's terms of service and
        privacy policy content. If this property is nil, the default @c FUIAuth instance's terms of service and 
        privacy policy will be used.
 */
@property(nonatomic, strong, nullable) FUIAuth *authUI;
```

Reply:
Done